### ISSUE TITLE
Plz add enigma2 webif

### ISSUE THREAD

**Issue by resToreHU** [iptv-org/epg#2324]
### Source

Http://webif

**Comment by HNJAMeindersma** [comment]
I don't think it would be possible to add this into the repo since Enigma2/OpenWebIf is usually a local hostname (inside a local area network). I'm also wondering if these receivers support exporting EPG data to a XMLTV format. Coincidentally I'll be experimenting with an OpenWebIf / OpenPLI / Enigma2 receiver in a couple of weeks.

**Comment by HNJAMeindersma** [comment]
I think you gonna need this: https://github.com/Jalle19/openwebif-xmltv-converter

**Comment by resToreHU** [comment]
k
Thank you for addressing the issue!

I would like it to be included in the EPG grabber project.

Webgab++ has already solved it, but it would be nice if it appeared here as well.
Enigma2 plugin working openpli 8.3 and openatv 7.3 and 7.4.

Project name: epgexport

Thanks

**Comment by HNJAMeindersma** [comment]
> I would like it to be included in the EPG grabber project.

What would be your proposal regarding host specification for a possible Enigma2 XMLTV grabber that fits in with the current state and input arguments of the repo?
